pkg/model/model.go

```diff
@@ -14,6 +14,14 @@ const (
 )
 
 const (
+	RelayAccountIDLabel           = "managed.relay.sh/account-id"
```

Comment:
Can we put some comments around these `managed` labels to indicate that they should be pulled out of this project once we have this fixed on the API side?

Reply:
Let me see how easy it is to just fix this instead. I'm not sure I want these to remain in the API itself, but this definitely isn't the place for them either.

Reply:
I think these should be defined and godoc'd as constants in the controllers that use them, but that's just my 0.02.